components/PostForm/PostForm.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useRef, useState, useEffect } from "react";
+import styles from "./PostForm.module.scss";
+import plusIcon from "public/icon_plus.svg";
+import xIconBlue from "public/icon_x-blue.svg";
+import Image from "next/image";
+
+const PostForm = () => {
+  const [title, setTitle] = useState("");
+  const [content, setContent] = useState("");
+  const [ImageFile, setProductImageFile] = useState<File | null>(null);
```

Comment:
```suggestion
  const [ImageFile, setImageFile] = useState<File | null>(null);
```
이름을 통일시켜주는 것이 더 보기 좋습니다!